src/features/emoji.rs

```diff
@@ -23,10 +27,10 @@ pub async fn add(params: CallBackParams<'_>) -> CallbackReturn {
       base64::encode(response_body)
     );
 
-    let guild = params
-      .message
-      .guild_id
-      .expect("Guildid wasn't found in the message");
+    let guild = match params.message.guild(params.context) {
+      Some(guild) => guild.id,
+      None => GuildId(GUILD_ID),
+    };
```

Comment:
Instead of match the or_default function is better for this default value use case